src/OpenCV_Mat.jl

```diff
@@ -41,27 +41,35 @@ inline void ptr_val3(cv::Mat &img, int row, int col, std::vector<double> vec)
 
 function pixget(img, row::Int, col::Int)
   (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
-
-  # Grayscale and binary images
-  val = @cxx ptr_val(img, row, col);
-  println(val)
-  return (val)
-
-  # RGB images
-  vec = @cxx ptr_val3(img, row, col);
-  println([int(at(vec, 0)), int(at(vec, 1)), int(at(vec, 2))])
-  return (vec)
+  image = Images.separate(img);
```

Comment:
@timholy 
Thanks for sharing the info here.  Is this documented in the `Images` project Wiki? It would be good to keep such relevant information in a Wiki. Perhaps we can have a new section either on `OpenCV.jl` or `Images.jl` that documents how to interact between these and other relevant graphics/image-processing Julia packages.

Or should we document such info in the JuliaImages [Wiki](https://github.com/JuliaImages/meta/wiki)?